src/TTextEdit.cpp

```diff
@@ -1207,7 +1248,7 @@ void TTextEdit::mousePressEvent( QMouseEvent * event )
             if ( xind > 0 ||
                  mpHost->mDoubleClickIgnore.contains(mpBuffer->lineBuffer[yind].at( xind ) ) )
                 mPA.setX ( xind+1 );
-            else
+            else if ( xind > 0 )
```

Comment:
What happens if `xind < 0` do we need to clamp that to 0 - perhaps by keeping the above as an `else` (or rather `else {` and making the next line(s):

        mPA.setX( qMax( 0, xind ) );
    }